<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cube3D</class>
 <widget class="QDialog" name="cube3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cube3D</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>0</y>
     <width>200</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 39);</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Alpha</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Phi</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDial" name="alphaDial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximum">
     <number>359</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>20</number>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="phiDial">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximum">
     <number>359</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>20</number>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="alphaLCD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Mode::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::SegmentStyle::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="phiLCD">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Mode::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::SegmentStyle::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinXC">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Arial&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinYC">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Arial&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinZC">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Arial&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Arial&quot;;

</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QDial" name="rotationAngleDial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximum">
     <number>359</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>20</number>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Rotation:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="rotationAngleLCD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Mode::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::SegmentStyle::Flat</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="xRotation">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>68</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 13pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yRotation">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>68</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 13pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="zRotation">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>68</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 13pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
